tests/compat.sh

```diff
@@ -216,14 +216,13 @@ filter_ciphersuites()
         G_CIPHERS=$( filter "$G_CIPHERS" )
     fi
 
-    # OpenSSL 1.0.1h doesn't support DTLS 1.2
-    if [ `minor_ver "$MODE"` -ge 3 ] && is_dtls "$MODE"; then
+    # OpenSSL <<1.0.2 doesn't support DTLS 1.2. Check what OpenSSL
```

Comment:
Isn't `OpenSSL <<1.0.2` a typo?

Reply:
[Habit](https://www.debian.org/doc/debian-policy/ch-relationships.html#syntax-of-relationship-fields), but I changed it to `<` by amending the last commit.

Reply:
I'm afraid that you didn't push the commit.